src/test/java/org/broadinstitute/consent/http/service/DatasetServiceTest.java

```diff
@@ -406,7 +418,8 @@ public void testUpdateDatasetDataUseNonAdmin() {
         DataUse dataUse = new DataUseBuilder().setGeneralUse(true).build();
         try {
             datasetService.updateDatasetDataUse(u, 1, dataUse);
-            fail("Should have thrown an exception on datasetService.updateDatasetDataUse()");
+            fail(
+                "Should have thrown an exception on datasetService.updateDatasetDataUse()");
         } catch (IllegalArgumentException e) {
             assertTrue(true);
         }
```

Comment:
nit: should this be in an assertThrows?

Reply:
In theory, yes. I'm trying hard NOT to refactor and instead doing as little as possible to make this functional under junit 5